### PR TITLE
maint: modifying files to conform with "PTH" ruff rule

### PR DESCRIPTION
**Pull request by moe-ad**
This PR is associated with #296. Flake9-use-pathlib ruff rule will be addressed via this PR.

### CODE REVIEW

**Comment by moe-ad** [comment]
Now all changes so far are passing the tests.

**Review by jorgepiloto — APPROVED** [review]
Thanks for such a big contribution, @moe-ad. This helps a lot with code quality and makes the project more robust.

**Comment by moe-ad** [comment]
Thank you too @jorgepiloto .

File: examples/08-python-operators/02-python_operators_with_dependencies.py [hunk 1]
```diff
@@ -58,11 +58,12 @@
...
 
 import os
+from pathlib import Path
 
 from ansys.dpf.core import examples
 
-plugin_path = examples.download_gltf_plugin()
-folder_root = os.path.join(os.getcwd().rsplit("pydpf-core", 1)[0], "pydpf-core")
+plugin_path = Path(examples.download_gltf_plugin())
+folder_root = Path(__file__).parent.parent.parent
```
Comment (jorgepiloto) [inline examples/08-python-operators/02-python_operators_with_dependencies.py:66]: This line is the one making the CI/CD to fail. I think the problem is that, this is converted to a notebook by Sphinx gallery and thus, the variable `__file__` is no longer present, @moe-ad

**Pull request merged**